dask/base.py

```diff
@@ -648,7 +649,10 @@ def tokenize(*args, **kwargs):
     """
     if kwargs:
         args = args + (kwargs,)
-    return md5(str(tuple(map(normalize_token, args))).encode()).hexdigest()
+    time_taken, result = get_wall_time(lambda: tuple(map(normalize_token, args)))
+    if time_taken > 2: # seconds
+        warnings.warn("tokenize is slow: set name=False to avoid hashing")
+    return md5(str(result).encode()).hexdigest()
```

Comment:
Why don't we want to include `md5`, `str`, ... in the hashing timing?

Reply:
I think they should be included.